.github/workflows/spm.yml

```diff
@@ -0,0 +1,17 @@
+name: spm
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  swift-build-run:
+    runs-on: macOS-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Xcode 12
+      run: sudo xcode-select -s /Applications/Xcode_12_beta.app/Contents/Developer
```

Comment:
this no longer seems necessary …

Reply:
@paulb777 , i don't know if adding spm.yml testing really is a complete non-starter.  the only part that i thought was "unnecessary" was changing `xcode-select` to use the -beta .

fwiw, we are at the place where we need to move to SPM for OCMock for some changes we are making this week, and may be interested in going with your fork as a starting point, especially if it just works out of the box for us.

Reply:
As I mentioned early on in this discussion (and repeated earlier today), OCMock uses Travis CI for continuous integration. Please do not add another CI system.